src/main/eo/org/eolang/collections/list.eo

```diff
@@ -49,23 +49,37 @@
   # for the index and the third one for the element
   # of the array.
   [a f] > reducedi
-    rec-reduced (* a 0) f arr > @
+    if. > @
+      eq.
```

Comment:
We can make it more compact:
```
if. > @
  (arr.length).eq 0
  a
  rec-reduced (* a 0) f arr
```
As for me, it reads better

Reply:
@MikhailLipanin I think we're trying to stick to vertical notation, @yegor256 correct me if I'm wrong.